qcodes/dataset/legacy_import.py

```diff
@@ -65,14 +72,21 @@ def store_array_to_database_alt(meas, array):
     return datasaver.run_id
 
 
-def import_dat_file(location: str) -> List[int]:
+def import_dat_file(location: str,
+                    exp: Optional[Experiment] = None) -> List[int]:
     """
-    This imports a QCoDeS legacy DataSet
+    This imports a QCoDeS legacy DataSet into the database.
+
+    Args:
+        location: Path to file containing legacy dataset
+        exp: Specify the experiment to store data to.
+            If None the default one is used.
```

Comment:
perhaps not "default" but "latest" or just refer to the docs of "Measurement" object?

Reply:
The default may change, but sure we can link to the measurement object.

Reply:
The docs of measurement are equally bad :( I will link that and fix it there